packages/patternfly-3/patternfly-react-extensions/less/table-grid.less

```diff
@@ -92,7 +100,47 @@
     .table-grid-pf-body {
       .row {
         cursor: pointer;
+      }
+    }
+  }
 
+  &.checkbox-select {
+    .table-grid-pf-checkbox {
+      display: initial;
+    }
+
+    .table-grid-pf-head {
+      position: relative;
+      padding-left: 35px;
+
+      .table-grid-pf-checkbox {
+        .btn {
+          color: @color-pf-blue-400;
+          font-size: 14px;
+          left: -1px;
+          padding: 0;
+          position: relative;
+          top: -2px;
```

Comment:
The only thing that bugs me about the design (and this may be on purpose) is that the border between the checkbox and the column header text in the header-row isn't vertically centered. Try removing `top: -2px;` and set `line-height: 1;` on this `.btn` - that looks centered to me.